org.ektorp/src/main/java/org/ektorp/impl/StdObjectMapperFactory.java

```diff
@@ -47,7 +47,7 @@ public void setWriteDatesAsTimestamps(boolean b) {
 
 	private void applyDefaultConfiguration(ObjectMapper om) {
 		om.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, this.writeDatesAsTimestamps);
-		om.getSerializationConfig().withSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
My only hesitation is versioning. The tutorial has been broken for some time now because of this (see #230).

In the long-term, definitely. If we agree the `StdObjectMapperFactory` should not be responsible for `ObjectMapper` configuration outside of the Ektorp-specific `@DocumentReferences` stuff it does in the `EktorpJacksonModule`, we could also consider removing line 49.

A non-intrusive change would be for the framework to provide another implementation that always respects the configuration of the given `ObjectMapper`. 

```java
public class ImmutableObjectMapperFactory implements ObjectMapperFactory {
    private final ObjectMapper objectMapper;

    public ImmutableObjectMapperFactory() {
        this(new ObjectMapper());
    }

    public ImmutableObjectMapperFactory(ObjectMapper objectMapper) {
        this.objectMapper = objectMapper;
    }

    @Override public ObjectMapper createObjectMapper() {
        return objectMapper;
    }

    @Override public ObjectMapper createObjectMapper(CouchDbConnector connector) {
        ObjectMapper copy = objectMapper.copy();
        copy.registerModule(new EktorpJacksonModule(connector, copy));
        return copy;
    }
}
```

This has the bonus of not needing any synchronization. It also allows the developer to make global configuration like setting the way a `DateTime` is serialized in one place, and have it be respected everywhere.